asmcomp/cmmgen.ml

```diff
@@ -2067,7 +2077,9 @@ and transl_prim_1 env p arg dbg =
       let prim = match bi with
         | Pnativeint -> "nativeint"
         | Pint32 -> "int32"
-        | Pint64 -> "int64" in
+        | Pint64 -> "int64"
+        | Puint32 -> "uint32"
+        | Puint64 -> "uint64" in
```

Comment:
if you moved that `in` to its own line, the diff for `{u,}int128` would not have this disgraceful change to an existing line.